app/models/teacher_training_public_api/sync_check.rb

```diff
@@ -14,6 +14,10 @@ def self.last_sync
       Redis.current.get(LAST_SUCCESSFUL_SYNC)
     end
 
+    def self.updated_since
+      (Time.zone.parse(last_sync) - 2.hours)
```

Comment:
We want to offset by the length of time it takes to complete the sync. Is 2 hours enough?

Reply:
Yea easily. It normally takes about 10-15 atm without incremental sync